tools/ctl/ctl.go

```diff
@@ -207,7 +207,30 @@ var tui = &cobra.Command{
 			NPM:      npmreg.HTTPRegistry{Client: regclient},
 			PyPI:     pypireg.HTTPRegistry{Client: regclient},
 		}
-		butler := localfiles.NewButler(*metadataBucket, *logsBucket, *debugStorage, mux, localfiles.AssetStore)
+		var assetStoreFn func(runID string) (rebuild.LocatableAssetStore, error)
+		if *sharedAssetStore != "" {
+			u, err := url.Parse(*sharedAssetStore)
+			if err != nil {
+				log.Fatal(errors.Wrap(err, "parsing --shared-asset-store"))
+			}
+			if u.Scheme != "gs" {
+				log.Fatal("--shared-asset-store must be a gs:// URL")
```

Comment:
not sure we want this constraint. having a local one would be fine, no? if we want to put off adding it, let's add a TODO and an "unimplemented" message